api/HTMLInputElement.json

```diff
@@ -47,6 +47,53 @@
           "deprecated": false
         }
       },
+      "align": {
```

Comment:
Can you add "align" to input.json? `<input align=right type=image>` confirms it does something.